### PR TITLE
Get rid of reflect metadata

### PR DESCRIPTION
**Pull request by YLivay**
Very few changes to avoid inferring TS types in some props around the codebase where possible

### CODE REVIEW

File: src/claim/views/key/_bundle/bundle.ts [hunk 1]
```diff
@@ -12,13 +13,13 @@ import { Store } from '../../../store/index';
 	},
 })
 export default class AppKeyBundle extends Vue {
-	@Prop()
+	@Prop({ required: true })
```
Comment (hworld) [inline src/claim/views/key/_bundle/bundle.ts:16]: can't this be propRequired(Object)?

File: src/utils/polyfills.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
-import 'core-js/modules/es7.object.values';
 import 'core-js/modules/es7.object.entries';
-import 'reflect-metadata';
+import 'core-js/modules/es7.object.values';
```
Comment (hworld) [inline src/utils/polyfills.ts:2]: why is this line after now?

File: src/_common/scroll/inview/parent.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { ScrollInviewContainer } from './container';
 export class AppScrollInviewParent extends Vue {
 	// If this is a child of AppScrollScroller, we need to get it as a prop so
 	// we can use that scroll element as the root context.
-	@Prop() scroller?: AppScrollScrollerTS;
+	@Prop({ required: false }) scroller?: AppScrollScrollerTS;
```
Comment (hworld) [inline src/_common/scroll/inview/parent.ts:10]: propRequired(Object)

File: package.json [hunk 4]
```diff
@@ -63,7 +63,6 @@
 		"date-fns": "1.30.1",
 		"decompress-zip": "0.3.1",
 		"express": "4.17.1",
-		"extract-loader": "3.0.0",
```
Comment (hworld) [inline package.json:66]: Why is this removed in this PR? Just wanted to clean up? Are you 100% sure we don't use it?

File: src/utils/polyfills.ts (hunk 2 above)
Comment (YLivay) [reply src/utils/polyfills.ts:2]: automatic import ordering.

File: package.json (hunk 4 above)
Comment (YLivay) [reply package.json:66]: yeah, this hasnt been used for ages now.

File: src/utils/polyfills.ts (hunk 2 above)
Comment (hworld) [reply src/utils/polyfills.ts:2]: it shouldn't reorder these raw imports....
